codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java

```diff
@@ -188,6 +190,8 @@ public void testResponseDecompression() {
     @Test
     public void testResponseBrotliDecompression() throws Throwable {
         Brotli.ensureAvailability();
+        // Failing on windows atm
+        Assume.assumeFalse(PlatformDependent.isWindows());
```

Comment:
@slandelle @hyperxpro these two tests fail on windows... I wonder if the native code not works as expected on windows.

Reply:
Does the CI have Visual C++ redists?

Reply:
no idea... but the failures is really an assert failure:

```
2021-05-19T16:21:56.7287182Z [ERROR] Failures: 
2021-05-19T16:21:56.7289070Z [ERROR]   HttpContentDecoderTest.testResponseBrotliDecompression:207 Content-Length header should match uncompressed string's length expected:<97715> but was:<92212>
2021-05-19T16:21:56.7292065Z [ERROR]   HttpContentDecoderTest.testResponseChunksBrotliDecompression:250 Content-Length header should match uncompressed string's length expected:<97715> but was:<92212>
2021-05-19T16:21:56.7293833Z [INFO] 
2021-05-19T16:21:56.7294428Z [ERROR] Tests run: 855, Failures: 2, Errors: 0, Skipped: 0
```

Reply:
Alright, not a dependency issue. I will investigate this and later do PR for enabling Brotli4j on Windows CI. Is that fine?

Reply:
yep... I will just skip these on windows for now.

Reply:
@hyperxpro any idea yet ?

Reply:
Unfortunately no. It's working fine cross-platform for me.

@slandelle How you compressed that data?